scylla/src/transport/session.rs

```diff
@@ -1231,12 +1239,10 @@ fn calculate_token(
         Err(PartitionKeyError::ValueTooLong(values_len)) => {
             return Err(QueryError::BadQuery(BadQuery::ValuesTooLongForKey(
                 values_len,
-                u16::max_value().into(),
+                u16::MAX.into(),
```

Comment:
This change should be placed into a separate commit, and maybe even into separate PR as it doesn't seem related.